ui/raidboss/data/triggers/o6s.js

```diff
@@ -4,21 +4,25 @@
   timelineFile: 'o6s.txt',
   triggers: [
     {
-      id: 'O6S Thing',
-      regex: /:1EDD:Alte Roite starts using/,
-      infoText: function(data, matches) {
-      },
-      tts: function(data, matches) {
+      id: 'O6S Demonic Stone',
+      regex: /1B:........:(\y{Name}):....:....:0001:0000:0000:0000:/,
+      alarmText: function(data,matches) {
+        if (data.me == matches[1])
+	  return 'Demonic Stone on YOU';
       },
     },
     {
-      id: 'O6S Headmarker',
-      regex: /1B:........:(\y{Name}):....:....:00XX:0000:0000:0000:/,
-      condition: function(data, matches) { return matches[1] == data.me; },
-      infoText: function(data) {
+      id: 'O6S Last Kiss',
+      regex: /1B:........:(\y{Name}):....:....:0017:0000:0000:0000:/,
+      alarmText: function(data,matches) {
+        if (data.me == matches[1])
+	  return 'Last Kiss on YOU';
       },
-      tts: function(data, matches) {
+      infoText: function(data,matches) {
+        if (data.me != matches[1])
+	return 'Last Kiss on ' +data.ShortName(matches[1]);
       },
+      tts: 'last kiss',
     },
   ]
-}]
+}
```

Comment:
You're missing a ] here.  This should be }], or raidboss will give you an error when you load this file.